packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1340,6 +1347,13 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
     ));
   }
 
+  // Unsupported
+  // TODO(MxSoul): see https://github.com/flutter/flutter/issues/100748
```

Comment:
same

Reply:
Because these are two different scenarios, and they're not coupled, we can probably solve them one by one